content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -145,11 +145,109 @@ TODO.
 
 TODO.
 
-## Gizmo configuration
-
-<div class="release-feature-authors">authors: @TODO</div>
-
-TODO.
+## Multiple gizmo configurations
+
+<div class="release-feature-authors">authors: @jeliag</div>
+
+Since [the 0.11 release], bevy supports gizmos. Gizmos allow drawing shapes using
+an immediate mode API. Here is how you use them:
+
+```rust
+// bevy 0.12.1
+fn set_gizmo_width(mut config: ResMut<GizmoConfig>) {
+    // set the line width of every gizmos with this global configuration resource.
+    config.line_width = 5.0;
+}
+
+fn draw_circles(mut gizmos: Gizmos) {
+    // Draw two circles with a 5 pixels outline
+    gizmos.circle_2d(vec2(100., 0.), 120., Color::NAVY);
+    gizmos.circle_2d(vec2(-100., 0.), 120., Color::ORANGE);
+}
+```
+
+Add a [`Gizmos`] system param and call a few methods, nothing more. Cool!
+
+Gizmos are also great for crate authors, they can use the same API.
+For example, the [`oxidized_navigation`] navmesh library uses gizmos for its debug overlay.
+Great!
+
+This is why gizmos were quickly adopted by the community.
```

Comment:
I think this sentence can be cut entirely and the meaning is still there with better flow.
```suggestion
```